src/index.html

```diff
@@ -144,6 +153,43 @@ <h2 class="modal__title">New Post</h2>
         <p class="modal__caption"></p>
       </div>
     </div>
+    <div class="modal" id="avatar-modal">
+      <div class="modal__container">
+        <div class="modal__header">
+          <button type="submit" class="modal__cls-btn modal__cls-btn_type_preview"></button>
```

Comment:
* Please, make the `avatar` modal match the design https://snipboard.io/YgOiWC.jpg . It should have the same styles like the other **form modals**


`modal__cls-btn_type_preview` is not needed for the avatar modal.


```html

          <button type="submit" class="modal__cls-btn"></button>

```